sparse_strips/vello_common/src/recording.rs

```diff
@@ -114,6 +120,38 @@ pub enum RenderCommand {
     SetPaintTransform(Affine),
     /// Reset the paint transform.
     ResetPaintTransform,
+    /// Fill glyphs with current paint.
+    #[cfg(feature = "text")]
+    FillGlyphs {
+        /// Font to use for the glyphs.
+        font: Font,
+        /// Font size in pixels per em.
+        font_size: f32,
+        /// Per-glyph transform.
+        glyph_transform: Option<Affine>,
+        /// Whether font hinting is enabled.
+        hint: bool,
+        /// Normalized variation coordinates for variable fonts.
+        normalized_coords: Vec<NormalizedCoord>,
+        /// The glyphs to fill.
+        glyphs: Vec<Glyph>,
+    },
+    /// Stroke glyphs with current paint.
+    #[cfg(feature = "text")]
+    StrokeGlyphs {
+        /// Font to use for the glyphs.
+        font: Font,
+        /// Font size in pixels per em.
+        font_size: f32,
+        /// Per-glyph transform.
+        glyph_transform: Option<Affine>,
+        /// Whether font hinting is enabled.
+        hint: bool,
+        /// Normalized variation coordinates for variable fonts.
+        normalized_coords: Vec<NormalizedCoord>,
+        /// The glyphs to stroke.
+        glyphs: Vec<Glyph>,
+    },
```

Comment:
Would it make sense to:

- Merge these two variants
- Add a `Style` field to disambiguate fill vs. stroke

?